tests/test_timeline_startup.py

```
@@ -0,0 +1,50 @@
+
+`main.py` does `from timeline_server import app` for the Railway timeline
+service, so a check that lives only in `timeline_server.main()` never runs
+on the deployment that most needs it.
+"""
+
+import importlib
+
+import pytest
+
+
+def _reload_server(monkeypatch, tmp_path):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_reload_server`

(ANN202)